RPi3.md

```diff
@@ -231,6 +231,8 @@ export BTCPAYGEN_ADDITIONAL_FRAGMENTS="opt-save-storage-xs;opt-save-memory"
 export BTCPAY_ENABLE_SSH=true
 ```
 
+Adding the `opt-save-storage-xs` fragment will keep around 3 months of blocks (prune BTC for 25 GB), and is necessary for FastSync to work. See other pruning levels [here](https://github.com/btcpayserver/btcpayserver-docker#generated-docker-compose-). 
```

Comment:
You need pruning for FastSync to work. But `-xs` not mandatory.

Reply:
Thanks for confirming Nico.

@o2C7YSae in that case could you just re-phrase to make it more clear, since the example above gives impression that -xs is mandatory, where instead any pruning option should work.

Reply:
Note that I can verify this in the load script...

Reply:
@NicolasDorier sorry, what load script? I'm not sure what you mean.